src/SFA.DAS.AANHub.Application/Apprentices/Commands/PatchApprenticeMember/PatchApprenticeMemberCommandResponse.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.AANHub.Application.Apprentices.Commands.PatchApprenticeMember
+{
+    public class PatchApprenticeMemberCommandResponse
+    {
+        public bool Success { get; set; }
```

Comment:
Remove set

Reply:
Didn't like it as it is being assigned in the below screenshot
![image](https://user-images.githubusercontent.com/46343852/217833975-e706c50a-8ebd-4bfe-a554-c6d5884d0308.png)

Reply:
You are assigning the same property twice, once my passing to ctor param and then immediately in object initialisation. You removed object init here.